src/foam/nanos/fs/File.js

```diff
@@ -267,7 +279,8 @@ foam.CLASS({
       name: 'authorizeOnDelete',
       javaCode: `
         AuthService auth = (AuthService) x.get("auth");
-        if ( ! auth.check(x, "file.remove." + getId()) ) {
+
+        if ( ! auth.check(x, "file.remove.id") ) {
```

Comment:
should be `"file.remove." + getId()`, will also want to allow users to remove the files they upload and own.

Reply:
okay, thank you.